### ISSUE TITLE
Crash on startup (Python3.8)

### ISSUE THREAD

**Issue by joex19x84** [PhotoFilmStrip/PFS#68]
I get this error when running:
```
Traceback (most recent call last):
  File "/usr/bin/photofilmstrip", line 8, in <module>
    main()
  File "/usr/lib/python3.8/site-packages/photofilmstrip/GUI.py", line 46, in main
    guiApp.Start()
  File "/usr/lib/python3.8/site-packages/photofilmstrip/AppMixin.py", line 41, in Start
    self.InitI18N()
  File "/usr/lib/python3.8/site-packages/photofilmstrip/AppMixin.py", line 31, in InitI18N
    ActionI18N().Execute()
  File "/usr/lib/python3.8/site-packages/photofilmstrip/action/ActionI18N.py", line 43, in Execute
    lang.install(True)
  File "/usr/lib/python3.8/gettext.py", line 352, in install
    for name in allowed & set(names):
TypeError: 'bool' object is not iterable

```

**Comment by fcole90** [comment]
Hi, this is still an issue in the version available in Ubuntu 20.04 and from the DEB package provided in the website.

**Comment by elizeubarbosaabreu** [comment]
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/wx/core.py", line 3282, in <lambda>
    lambda event: event.callable(*event.args, **event.kw) )
  File "/usr/lib/python3/dist-packages/photofilmstrip/lib/common/ObserverPattern.py", line 26, in Notify
    observer.ObservableUpdate(self, arg)
  File "/usr/lib/python3/dist-packages/photofilmstrip/gui/ImageSectionEditor.py", line 92, in ObservableUpdate
    self.__Scale()
  File "/usr/lib/python3/dist-packages/photofilmstrip/gui/ImageSectionEditor.py", line 122, in __Scale
    self._imgProxy.Scale(newWidth, newHeight)
  File "/usr/lib/python3/dist-packages/photofilmstrip/gui/ImageSectionEditor.py", line 675, in Scale
    img = self._wxImg.Scale(width, height)
TypeError: Image.Scale(): argument 1 has unexpected type 'float'
